eslint.config.js

```diff
@@ -0,0 +1,59 @@
+import hypothesis from 'eslint-config-hypothesis';
+import jsxA11y from 'eslint-plugin-jsx-a11y';
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  {
+    ignores: ['lib/**', 'lib-cjs/**', 'build/**'],
```

Comment:
`lib-cjs` doesn't exist anymore after the CommonJS build was removed in 9acbc98f13d86451b1a7520e464a72078c7dae4e.